.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,42 @@
+name: release-binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag_name:
+        required: true
+        type: string
+        description: >
+          The tag that a release has been previously created from. Binaries will be attached to this release.
+  workflow_call:
+    inputs:
+      release_tag_name:
+        required: true
+        type: string
+        description: >
+          The tag that a release has been previously created from. Binaries will be attached to this release.
+
+jobs:
+  release:
+    runs-on: macos-latest-xl
```

Comment:
12-core macOS runner 🚀 https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources